### PR TITLE
Use the fabric8 java-generator instead of committing a jar

### PR DESCRIPTION
**Pull request by andreaTP**
I haven't tested those changes locally, but happy to collaborate if any issues arise.

### CODE REVIEW

**Comment by andreaTP** [comment]
cc. @kornys

**Review by kornys — APPROVED** [review]

**Comment by kornys** [comment]
I triggered pipeline with this changes and seems it works, but @henryZrncik please check it as well

**Comment by andreaTP** [comment]
@kornys as agreed used the builder pattern with the generated code.

@henryZrncik this should be ready for review now, thanks in advance! 🙂

File: pom.xml [hunk 1]
```diff
@@ -280,6 +288,78 @@
...
+                    <phase>generate-sources</phase>
+                    <goals>
+                      <goal>generate</goal>
+                    </goals>
+                  </execution>
+                </executions>
+                <configuration>
+                  <source>${project.build.directory}/manifests</source>
+                  <extraAnnotations>true</extraAnnotations>
+                </configuration>
```
Comment (henryZrncik) [inline pom.xml:361]: just a little different indentation from rest of the file.

**Review by henryZrncik — APPROVED** [review]
great work, looks good. 
Thanks for simplification of work with these models.

File: pom.xml (hunk 1 above)
Comment (andreaTP) [reply pom.xml]: should be fixed now 👍

**Pull request merged**